### DISCUSSION TITLE
error saat knit untuk flexdashboard

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#75] [Q&A] [answered]
Kenapa error ya waktu knit di flexdashboard ? Tapi waktu buat plot ngga ada masalah.

<img width="658" alt="image" src="https://user-images.githubusercontent.com/103427989/165965390-f41a702f-573f-45cc-9a7b-66592cf24c5c.png">

**Comment by dwiknrd** [comment, marked as answer]
Halo Pak [joejnt](https://github.com/joejnt)

Pastikan tidak ada object yang terpanggil sebelum anda melakukan assignment kepada dengan nama object tersebut. 

Apabila dilihat dari message error, error code terdapat pada line 22 yang mana ada object bernama `casatd` yang belum didefinisi. Dan apabila dilihat pada line 39 anda melakukan assignment pada object `casatd`.

Semoga bermanfaat 😊
